xaitk_saliency/utils/gen_coco_sal.py

```diff
@@ -55,6 +56,11 @@ def gen_coco_sal(
         :param img_perturber: Implementation of ``PerturbImage`` to use.
         :param sal_generator: Implementation of
             ``GenerateDetectorProposalSaliency`` to use.
+        :param fill: Optional fill for the occluded regions of each image as a
+            scalar value, a per-channel sequence or a shape-matched image. If
+            a multi-channel value is used, all the images in ``dets_dset`` must
+            have that number of channels. Similarly, if a shape-matched image
+            is used, it must match the shape of each image in ``dets_dset``.
```

Comment:
Since we are passing this through to the `occlude_image_batch` function, should the `fill` parameter be synced to the docstring there? We mention alpha-blending (I think this was used for blurring), although I'm not 100% sure it is applicable here, e.g.

```suggestion
        :param fill: Optional fill for alpha-blending based on the input masks for
            the occluded regions as a scalar value, a per-channel sequence or a
            shape-matched image. If a multi-channel value is used, all the images
            in ``dets_dset`` must have that number of channels. Similarly, if a
            shape-matched image is used, it must match the shape of each image
            in ``dets_dset``.
```

Reply:
Actually, thinking more about this, I think the tool can probably only support occlusion, i.e. the shape-matched fill value doesn't make sense here? Our CLI tool probably also doesn't support passing in images.

Reply:
This description is in `gen_coco_sal`, the programmatic tool. If you are using this just as a function in a program you can pass an image.